client/src/components/AddNewProfile.js

```diff
@@ -35,6 +35,11 @@ const AddNewProfile = () => {
     updateField(data.name, data.value);
   };
 
+  const groupsHandler = (event, data) => {
+    const selectedGroups = data.value;
+    profileData.groups = selectedGroups;
```

Comment:
```suggestion
    setProfileData({ ...profileData, groups: selectedGroups });
```
`profileData` is from `useState`, and whenever we update state we should always use the setter function so that React knows to re-render

Reply:
thanks @bonboh